packages/nova-react-test-utils/README.md

```diff
@@ -125,7 +124,7 @@ export const LikeFailure: Story = {
 };
 ```
 
-This time not resolvers are queued up front so inside [play](https://storybook.js.org/docs/react/writing-stories/play-function#page-top) one needs to manually resolve/reject graphql operations. To get the environment created for this specific story one can use `getEnvForStory` function. Later similarly to examples for unit test, full customization power of apollo-mock-client is available.
+This time not resolvers are queued up front so inside [play](https://storybook.js.org/docs/react/writing-stories/play-function#page-top) one needs to manually resolve/reject graphql operations. To get the environment created for this specific story one can use `getNovaEnvironmentDecorator` function. Later similarly to examples for unit test, full customization power of apollo-mock-client is available.
```

Comment:
> This time **all** resolvers …

?

Reply:
Ah just read in context of the whole document, so it should be:

> This time **no** resolvers

Reply:
Fixed